client/package.json

```diff
@@ -11,7 +11,10 @@
     "@types/react": "^16.9.0",
     "@types/react-dom": "^16.9.0",
     "@types/react-router-dom": "^5.1.6",
+    "bootstrap": "^4.5.3",
```

Comment:
Something good to know, in the client (UI) code, general practice is to remove the ^ from the dependencies so that everyone has the same versions. The ^ arrow means when a dev runs yarn install, it will look for the newest version of that package and auto-upgrade the dependency. I don't think this is a big deal for us since we are such a short time frame for our project but just fyi :D